updating/updating-cluster.adoc

```diff
@@ -7,6 +7,18 @@ toc::[]
 
 You can update, or upgrade, an {product-title} cluster.
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
```

Comment:
* This mentions beta without a version. Should it say `During the {product-title} 4 beta period` ?
* Consider leading in to the bullets like "update functionality has the following limitations" or "update functionality is limited in the following ways"